syncrclone/cli.py

```diff
@@ -294,8 +294,10 @@ def cli(argv=None):
     except Exception as E:
         import tempfile
         
+        tempfile.tempdir = os.path.join(tempfile.gettempdir(), config.tmp_dir_suffix)
```

Comment:
Few things:

* This is only in the case of an error. Does it need to change?
* Whereas the original had a directory that never existed, this one may. In that case, I'd much rather [EAFP vs LBYL](https://devblogs.microsoft.com/python/idiomatic-python-eafp-versus-lbyl/)

Reply:
Also, this may not work on macOS with a different tempdir semantics. See my other comment for a way to do this